### ISSUE TITLE
Session error with stateless firewall - configuration options 

### ISSUE THREAD

**Issue by jmiridis** [auth0/symfony#155]
### Checklist

- [X] I have looked into the [Readme](https://github.com/auth0/symfony#readme) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/symfony/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

With Symfony feature[ #48044,](https://github.com/symfony/symfony/commit/cb6f2c58ff68662a6616d11a084ba72844f2a5d8) a firewall that is configured stateless will add an attribute `_stateless` to the request. 
The AbstractSessionListener checks the attribute at the end of a request and throws an UnexpectedSessionUsageException("Session was used while the request was declared stateless.") if a session has been started.
I am using the `auth0.authorizer` authorizer and `stateless: true` in my firewall which now throws this error.

### Describe the ideal solution

I am not sure whether this is a bug or something that could be handled by allowing a different configuration. Is there a way to change the configuration to prevent this error or could it be added?

**Comment by evansims** [comment]
Hey @jmiridis 👋 Thanks for raising this.

This appears to be a new feature introduced into Symfony after our most recent SDK release. I'll need to investigate this a bit more and get back to you.

**Comment by evansims** [comment]
Hi @aless673, thanks for reporting. We do not presently support the stateless configuration property — it would have been purely coincidental if it worked in any previous version. It’s on our radar.

**Comment by DavidBadura** [comment]
I have the same problem. Does anyone have a workaround?

**Comment by jvdhehvg** [comment]
> I have the same problem. Does anyone have a workaround?

@DavidBadura you can define a strategy in the auth0 config like so:

```
auth0:
    sdk:
        ...
        strategy: "api"
        ...
```

This will make sure that the SDK will not start a session. 
Does this solve your problem?

**Comment by jasonspriggs** [comment]
Not sure if this is the issue you were running into, but I was getting a 500 error back with a log to the effect of:

```
[critical] Uncaught PHP Exception Error: "Call to undefined function Auth0\SDK\Mixins\mb_regex_encoding()" at /app/vendor/auth0/auth0-php/src/Mixins/ConfigurableMixin.php line 205
```

I dug into the error a bit and went down an incorrect rabbit hole trying to see what I did wrong. Turns out, the answer is a bit more obvious. The upstream library, [auth0-php](https://github.com/auth0/auth0-PHP), does not specify mbstring, a PHP extension, as a requirement. So if your system doesn't have that installed, it throws that error. In my case, I'm running an application on Heroku, which requires you to `composer require ext-mbstring` to enable that extension in the container.
